cmd/harmony/bls.go

```diff
@@ -25,7 +27,19 @@ func setupConsensusKeys(hc harmonyConfig, config *nodeconfig.ConfigType) multibl
 			os.Exit(100)
 		}
 	})
-	config.ConsensusPriKey = multiBLSPriKey
+
+	// Dedup private keys
+	uniqueKeys := map[bls.SerializedPublicKey]struct{}{}
```

Comment:
Do you think the readability of this code can be optimized? What about refactoring this logic into a separate function like `dedupPrivateKeys`, and then add the logic to the end of `loadBLSKeys` (cmd/harmony/bls.go:46)?

Also please use 

```
m := make(map[bls.SerializedPublicKey]struct{})
```

for nil map initialization as a convention. ([Reference](https://blog.golang.org/maps))

Reply:
Done.